README.md

```diff
@@ -7,7 +7,23 @@ This is all the source code used to create and run the IQ Control Center.
 ## Getting Started
 
 If you just want to use the IQ Control Center please follow this link to the latest [release](https://github.com/iq-motion-control/iq-control-center/releases) where you will find precompiled versions of the IQ Control Center for Windows, Mac, and Linux.
-  
+
+### General Installation Instructions
```

Comment:
The only change made is to the instructions so people can more easily use our Linux releases.